### ISSUE TITLE
py pkg pyqt5 - Error: wrapQtAppsHook is not used, and dontWrapQtApps is not set.

### ISSUE THREAD

**Issue by InLaw** [DavHau/mach-nix#379]
current 3.3.0

e.g. spyder-notebook 

```
Error: wrapQtAppsHook is not used, and dontWrapQtApps is not set.
builder for '/nix/store/z7rb413064lxqw1iygzrg1d5nrq60v6v-python3.8-pyqt5-5.12.3.drv' failed with exit code 1
```

**Comment by exarkun** [comment]
I've worked around this in other packages by including:

```
        # Get qtWrapperArgs set, even though wrapQtAppsHook won't wrap the                                                                                                        
        # gridsync executable for us.                                                                                                                                             
        nativeBuildInputs = [ pkgs.qt5.wrapQtAppsHook ];

        # For some reason pyqt5 fails to build like this if we don't add this                                                                                                     
        # hook:                                                                                                                                                                   
        #                                                                                                                                                                         
        #  Error: wrapQtAppsHook is not used, and dontWrapQtApps is not set.                                                                                                      
        #                                                                                                                                                                         
        _.pyqt5.nativeBuildInputs.add = [ pkgs.qt5.wrapQtAppsHook ];

        # Wrap it ourselves because non-ELF executables are ignored by the                                                                                                        
        # automatic wrapping logic.  This gets the Qt environment properly                                                                                                        
        # initialized.                                                                                                                                                            
        makeWrapperArgs = [
          "\${qtWrapperArgs[@]}"
          # import Qt.labs.platform failed without this                                                                                                                           
          # "--prefix QML2_IMPORT_PATH : ${qt5.qtquickcontrols2.bin}/${qt5.qtbase.qtQmlPrefix}"                                                                                   
        ];
```

**Comment by DavHau** [comment]
fixed via f0f48efdd6a1bc6d7834e822f2f364e8433e75e8
